### PR TITLE
Java SDK Implementation

### PR DESCRIPTION
**Pull request by erzohrakhan**
This PR addresses issue #251
- Added barebone structure of spring boot app
- Thymeleaf dependency, Maven Spotless plugin, Hibernate validator are added

### CODE REVIEW

File: java/configu-sdk/pom.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	<name>configu-sdk</name>
+	<description>Configu SDK</description>
+	<properties>
+		<java.version>17</java.version>
+		<spring-shell.version>3.1.3</spring-shell.version>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-thymeleaf</artifactId>
```
[the remark this answers is not in the document]
Comment (erzohrakhan) [reply java/configu-sdk/pom.xml:23]: Thymeleaf is a powerful choice for Java-based web applications, especially when integrated with the Spring Framework. Mustache, on the other hand, is a lightweight, logic-less template engine suitable for various programming languages and scenarios where simplicity and maintainability are prioritized.

**Review by Sadaf-A — APPROVED** [review]

**Comment by RichardAkman** [comment]
@erzohrakhan @Sadaf-A, we've collectively decided to change the direction of this issue by focusing on a minimalistic Java SDK implementation, hence the name change. There's no need to involve the Java Spring framework for running the SDK, as it's unnecessary and could potentially add unnecessary complexity for future contributions. Please ensure that the implementation remains devoid of the Spring framework and stays lightweight 🙏🏻

**Comment by Sadaf-A** [comment]
> @erzohrakhan @Sadaf-A, we've collectively decided to change the direction of this issue by focusing on a minimalistic Java SDK implementation, hence the name change. There's no need to involve the Java Spring framework for running the SDK, as it's unnecessary and could potentially add unnecessary complexity for future contributions. Please ensure that the implementation remains devoid of the Spring framework and stays lightweight 🙏🏻 

Definitely agree, @erzohrakhan what do you think?

**Comment by rannn505** [comment]
Hi @erzohrakhan @Sadaf-A 

Firstly, we want to express our gratitude for your initiative and the effort you've put into this Pull Request. Your contributions are truly appreciated.

However, we've noticed that there hasn't been any progress on this PR for over a month. To keep our project's development flow efficient and manageable, we'll be closing this PR for now. 

That said, we absolutely encourage and welcome future contributions from you. If you have something more concrete or when you're ready to resume work on this, please feel free to open a draft PR. We're always excited to see new ideas and progress.

Thanks again for your understanding and your contributions to the project. Looking forward to your future involvement!

**Pull request closed without merging**